pythonforandroid/recipes/cryptography/__init__.py

```diff
@@ -13,14 +13,9 @@ def get_recipe_env(self, arch):
         env = super(CryptographyRecipe, self).get_recipe_env(arch)
         r = self.get_recipe('openssl', self.ctx)
         openssl_dir = r.get_build_dir(arch.arch)
-        env['PYTHON_ROOT'] = self.ctx.get_python_install_dir()
-        env['CFLAGS'] += ' -I' + env['PYTHON_ROOT'] + '/include/python2.7' + \
-                         ' -I' + join(openssl_dir, 'include')
```

Comment:
Thanks for updating that pull request.
Please remove the `join` import since it's not used anymore Travis is complaining.
https://travis-ci.org/kivy/python-for-android/jobs/398189288
Also think about running `tox` so you don't have to wait for Travis to fail to fix the linter.